output/output.go

```diff
@@ -1,24 +1,39 @@
 package output
 
 import (
-	"encoding/json"
 	"fmt"
 
 	"github.com/codegangsta/cli"
 )
 
-// JSON prints results in JSON format.
-func JSON(i interface{}) {
-	j, _ := json.Marshal(i)
-	fmt.Println(string(j))
-}
-
 // Print prints the results of the CLI command.
-func Print(c *cli.Context, i interface{}, table func(*cli.Context, interface{})) {
+func Print(c *cli.Context, f *func() interface{}, keys []string) {
+	i := (*f)()
```

Comment:
The only place it would change would be for functions that return raw output like `keypair get`, for example.